cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -407,9 +412,14 @@ def accelerator_experiments(self, key, value):
     _extensions = self.get("extensions", {})
 
     sub_a = clean_val("a", value, str)
-    sub_e = clean_val("e", value, str)
+    sub_e = mapping(
+            EXPERIMENTS,
+            clean_val("e", value, str),
+            raise_exception=True,
+        )
     sub_p = clean_val("p", value, str)
 
+    # TODO: Add mapping for accelerators (Vocabulary)
```

Comment:
Is this still valid? If yes, is there an issue for this? Please prefer issues with a tag BLOCKER instead, we will forget all TODOs otherwise

Reply:
Yes, the librarians are cleaning the values, and when they finish we can add the mapping. From a previous comment from @kprzerwa  I understand that we will wait until the librarians clean the values to merge this PR, so we can add the mapping in this same PR.

Reply:
yes, could you please ask them when we could expect the values ?